### DISCUSSION TITLE
Does this mod affect frost cave generation?

### DISCUSSION THREAD

**Discussion by carpenteer** [Digitalroot-Valheim/Digitalroot.Valheim.MaxDungeonRooms#15] [Q&A] [answered]
The question is in the title.  I'm wondering if this mod is *already* making Frost Caves (potentially) bigger than vanilla.

**Comment by Digitalroot** [comment, marked as answer]
If they are created the same way as the other dungeons. Then yes.

**Comment by cawphee-black** [comment]
I wondered the same thing... either the dev got really ambitious in their size settings for Frost Caves or this mod is affecting the size because some of the frost caves I've explored are HUGE

**Reply by Digitalroot** [reply]
Is huge a good thing?

**Reply by carpenteer** [reply]
In my opinion?  Fuck yes!  The bigger the better, for dungeons and such!
